src/AppBundle/Repository/EventRepository.php

```diff
@@ -264,19 +264,17 @@ public function getRevisions(Event $event, ?int $offset = 0, ?int $limit = 50, b
         // Check cache and return if it exists, unless the Event was recently updated,
         // in which case we'll want to invalidate the cache.
         $shouldUseCache = null !== $event->getUpdated() &&
-            (int)$event->getUpdated()->format('U') < time() - $cacheDuration &&
-            null !== $this->getRedisConnection();
+            (int)$event->getUpdated()->format('U') < time() - $cacheDuration;
         $cacheKey = $this->getCacheKey(func_get_args(), 'revisions');
-        if ($shouldUseCache && $this->getRedisConnection()->contains($cacheKey)) {
-            return $this->getRedisConnection()->fetch($cacheKey);
+        if ($shouldUseCache && $this->cache->hasItem($cacheKey)) {
+            return $this->cache->getItem($cacheKey)->get();
         }
 
         $ret = $this->getRevisionsData($event, $offset, $limit, $count);
 
         // Cache for 5 minutes.
         if ($shouldUseCache) {
-            $redis = $this->getRedisConnection();
-            $redis->save($cacheKey, $ret, $cacheDuration);
+            return $this->setCache($cacheKey, $ret, 'PT5M');
```

Comment:
This method [`setCache`](https://api.symfony.com/4.1/Symfony/Component/HttpFoundation/Response.html#method_setCache) appears to be related to a `Response` object and not really the `Cache` system. Maybe we want something like [`save()`](https://api.symfony.com/4.1/Symfony/Component/Cache/Adapter/ApcuAdapter.html#method_save) instead?

Reply:
It probably could have a better method name, but this is from [Repository.php#L188](https://github.com/wikimedia/eventmetrics/blob/master/src/AppBundle/Repository/Repository.php#L188). It caches whatever you give it for the specified time, and returns the same thing (`mixed` type). So basically it's just a shorthand.